indexer-app/README.md

```diff
@@ -160,6 +160,12 @@ Make changes to the collection's rebalancing status. This will update a mapping
 
     curl -XPOST http://localhost:3004/index-sets/3/rebalancing-collections/C5-PROV1/update-status?status=COMPLETE
 
+### Start resharding an index
+
+An index can be resharded to improve performance. The `start` endpoint begin the process.
```

Comment:
It looks like from the code the resharding logic will only work for granule indexes. We'll want to be able to do this for any index (increase shards for variables, decrease for all the generics). We can handle other concept types as a separate ticket(s).